.github/workflows/nodejs-check-code.yml

```diff
@@ -1,18 +0,0 @@
-name: "Nodejs Check Code"
-
-on:
-  pull_request:
-    branches: [main]
-
-jobs:
-  check-code:
-    name: Nodejs Check Code
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 18
```

Comment:
The PR adds `check-code` but not with this node versions.
Since concourse shared syncs across all nodejs repos together, therefore only if all of our nodejs repos support node version 18, then we can make a change there and it should fix the failing tests.